GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="7e053cd8-2f6f-46f0-b7bb-b27448358047"
+    Publisher="CN=collabera"
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="7e053cd8-2f6f-46f0-b7bb-b27448358047" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>MyUWPGetStartApp</DisplayName>
+    <PublisherDisplayName>collabera</PublisherDisplayName>
```

Comment:
@nishitha-burman @peiche-jessica - change to microsoft? --

```
<PublisherDisplayName>microsoft</PublisherDisplayName>
```

todo: see what's in other project dirs .appmanifest files

Reply:
Changed to "Microsoft" 2x, built/ran ok.